lib/controller.js

```diff
@@ -733,9 +712,9 @@ controller.setWiFiState = function(opts) {
 controller.createAccessPoint = function(opts) {
   opts.authorized = true;
   var ssid = opts.ssid;
-  var password = opts.pass;
+  var password = opts.password;
   var security = opts.security;
-  var securityOptions = ['none', 'wep', 'psk', 'psk2', 'wpa', 'wpa2'];
+  var securityOptions = ['none', 'wep', 'psk', 'psk2'];
```

Comment:
Yes because Enterprise WPA requires a separate RADIUS authentication server to setup an access point -> https://wiki.openwrt.org/doc/uci/wireless#wpa_enterprise_access_point

Connecting to a Enterprise WPA network only requires some different configuration settings.